### PR TITLE
Adding the fix for issue: New workloadspec classes #1, and updating e…

### PR DESCRIPTION
**Pull request by GhaziSyed**

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -7,23 +7,23 @@ Update the relevant sections under `if location == "local"` or `if location == "
 
 ### Define and initalize the Model Specification
 There are three different ways to construct a model specification:
-1. By calling the constructor of `TGISModelSpec` or `vLLMModelSpec` directly (as defined in [this file](https://github.com/fmperf-project/fmperf/blob/main/fmperf/ModelSpecs.py)).
-2. By calling `TGISModelSpec.from_yaml` or `vLLMModelSpec.from_yaml` and passing a path to a YAML file defining the specification (see example [here](https://github.com/fmperf-project/fmperf/blob/main/examples/model_specifications_tgis_one.yml) for a `TGISModelSpec`).
+1. By calling the constructor of `TGISModelSpec` or `vLLMModelSpec` directly (as defined in [this file](/fmperf/ModelSpecs.py).
+2. By calling `TGISModelSpec.from_yaml` or `vLLMModelSpec.from_yaml` and passing a path to a YAML file defining the specification (see example [here](/examples/model_specifications_tgis_one.yml) for a `TGISModelSpec`). There is a one-to-one mapping between the constructor arguments and the key-value pairs in the yaml files.
 
 ### Define and initalize the Workload Specification
```
Comment (tdoublep) [inline examples/README.md:13]: This section title is repeated twice

File: examples/README.md [hunk 2]
```diff
@@ -7,23 +7,23 @@ Update the relevant sections under `if location == "local"` or `if location == "
 
 ### Define and initalize the Model Specification
 There are three different ways to construct a model specification:
```
Comment (tdoublep) [inline examples/README.md:9]: two different ways

File: examples/README.md [hunk 3]
```diff
@@ -7,23 +7,23 @@ Update the relevant sections under `if location == "local"` or `if location == "
...
-1. By calling the constructor of `TGISModelSpec` or `vLLMModelSpec` directly (as defined in [this file](https://github.com/fmperf-project/fmperf/blob/main/fmperf/ModelSpecs.py)).
-2. By calling `TGISModelSpec.from_yaml` or `vLLMModelSpec.from_yaml` and passing a path to a YAML file defining the specification (see example [here](https://github.com/fmperf-project/fmperf/blob/main/examples/model_specifications_tgis_one.yml) for a `TGISModelSpec`).
+1. By calling the constructor of `TGISModelSpec` or `vLLMModelSpec` directly (as defined in [this file](/fmperf/ModelSpecs.py).
```
Comment (tdoublep) [inline examples/README.md:10]: do these links resolve? haven't tested it myself

File: examples/README.md (hunk 1 above)
Comment (GhaziSyed) [reply examples/README.md:13]: Thanks for the review.
Not sure I follow this.

File: examples/README.md (hunk 2 above)
Comment (GhaziSyed) [reply examples/README.md:9]: Okay

File: examples/README.md (hunk 3 above)
Comment (GhaziSyed) [reply examples/README.md:10]: Yes. I changed to the links relative to the workspace directory, so the user can access these links on their development machine without going to github. They work

**Pull request merged**